processor/src/debug.rs

```diff
@@ -70,6 +71,11 @@ impl Iterator for VmStateIterator {
             } else {
                 Some(self.process.decoder.get_operation_at(self.clk - 1))
             },
+            asmop: if self.clk == 0 {
+                None
+            } else {
+                self.process.decoder.get_asmop_at(self.clk - 1)
+            },
```

Comment:
I found this to be a bit difficult to follow. Maybe define variables for `op` and `asmop` separately and then use these variables in construction of `result`?

Reply:
Done.